src/tokens-studio/foundations/$themes.json

```diff
@@ -0,0 +1,97 @@
+[
+  {
+    "id": "738333a3145b5fb23c20797c93ee780d037a11d5",
+    "name": "light",
+    "selectedTokenSets": {
+      "spectrum/palette/light": "enabled",
+      "spectrum/alias/light": "enabled",
+      "spectrum/component/light": "enabled",
+      "spectrum/icon/light": "enabled"
+    },
+    "group": "S.Color-theme"
+  },
+  {
+    "id": "51d38b3af725b927e3fba235fa6c3bb6cfcaf59e",
+    "name": "dark",
+    "selectedTokenSets": {
+      "spectrum/palette/dark": "enabled",
+      "spectrum/alias/dark": "enabled",
+      "spectrum/component/dark": "enabled",
+      "spectrum/icon/dark": "enabled"
+    },
+    "group": "S.Color-theme"
+  },
+  {
+    "id": "c9bc639cbf2113e23ab199cc67f78dd936c8d66a",
+    "name": "wireframe",
+    "selectedTokenSets": {
+      "spectrum/palette/wireframe": "enabled",
+      "spectrum/alias/wireframe": "enabled",
+      "spectrum/component/wireframe": "enabled",
+      "spectrum/icon/wireframe": "enabled"
+    },
+    "group": "S.Color-theme"
+  },
+  {
+    "id": "2083d14f7264328dfb689d5991f27a55c9069440",
+    "name": "mobile",
+    "$figmaStyleReferences": {},
+    "selectedTokenSets": {
+      "spectrum/icon/mobile": "enabled",
+      "spectrum/layout/mobile": "enabled",
+      "spectrum/layout.component/mobile": "enabled",
+      "spectrum/typography/mobile": "enabled",
+      "spectrum/typography.component/mobile": "enabled"
+    },
+    "group": "S.Scale"
+  },
+  {
+    "id": "12629b2474b904be31cfd0ec84ae76e92c88576e",
+    "name": "desktop",
+    "$figmaStyleReferences": {},
+    "selectedTokenSets": {
+      "spectrum/icon/desktop": "enabled",
+      "spectrum/layout/desktop": "enabled",
+      "spectrum/layout.component/desktop": "enabled",
+      "spectrum/typography/desktop": "enabled",
+      "spectrum/typography.component/desktop": "enabled"
+    },
+    "group": "S.Scale"
+  },
+  {
+    "id": "738333a3145b5fb23c20797c93ee780d037a11d5",
+    "name": "light",
+    "selectedTokenSets": {
+      "spectrum/opacity.alias/light": "enabled",
+      "spectrum/opacity.component/light": "enabled"
+    },
+    "group": "S.Color-theme"
+  },
+  {
+    "id": "51d38b3af725b927e3fba235fa6c3bb6cfcaf59e",
+    "name": "dark",
+    "selectedTokenSets": {
+      "spectrum/opacity.alias/dark": "enabled",
+      "spectrum/opacity.component/dark": "enabled"
+    },
+    "group": "S.Color-theme"
+  },
+  {
+    "id": "09fb335cab18186b281e7d5394bf56a0fa6e625f",
+    "name": "darkest",
```

Comment:
@mrcjhicks maybe this slipped through?

Reply:
I'll search and see if any others slipped through.

Reply:
Looks like that was the only place darkest was included, and it wasn't in any actual data, just this metadata.